docs/cli/templates.md

```diff
@@ -145,11 +145,86 @@ Now you can open your current folder in a tool such as Visual Studio Code and ad
 
 ## 5.0 Ruby
 
-OpenFaaS has first-class support for Ruby.
 
-Just create a new function by passing in `--lang ruby` as an argument.
+Create a function called `ruby-function`:
+
+```
+$ faas-cli new --lang ruby ruby-function
+```
+
+The directory structure is:
+
+```
+├── ruby-function
+│   ├── Gemfile
+│   └── handler.rb
+├── ruby-function.yml
+```
+
+Your code should be in the handler.rb file
+
+### 5.1 Adding a Gem (Library)
+
+Open the `Gemfile` in the ruby-function directory
+
+Add the following line
+
+```
+gem 'qrcoder'
```

Comment:
This should be `httparty` I think?

Reply:
I fixed that before you submitted your comment haha